File: Data Structures & Algorithms/longest-consecutive-sequence/submission-3.java
class Solution {
    public int longestConsecutive(int[] nums) {
        HashSet<Integer> set = new HashSet<>();

        for(int i: nums){
            set.add(i);
        }
        int mxSize =0;
        int size = 0;
        for(int i: nums){
            size = 0;
            var curr = i;
            while(set.contains(curr)){
                curr++; size++;
            }
            if(size > mxSize)
                mxSize = size;
        }
        return mxSize;
    }
}

File: Data Structures & Algorithms/find-duplicate-integer/submission-3.py
class Solution:
    def findDuplicate(self, nums: List[int]) -> int:
        for i in nums:
            print(nums, i)
            if nums[abs(i)-1] < 0:
                return abs(i)
            else:
                nums[abs(i)-1] *= -1
        
        return -9999